### PR TITLE
whipper: 0.7.0 -> 0.7.2

### PR DESCRIPTION
**Pull request by Moredread**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/audio/whipper/default.nix [hunk 1]
```diff
@@ -3,13 +3,13 @@
 
 python2.pkgs.buildPythonApplication rec {
   name = "whipper-${version}";
-  version = "0.7.0";
+  version = "0.7.2";
```
Comment (Mic92) [inline pkgs/applications/audio/whipper/default.nix]: The following should make the patch unnecessary:

```
makeWrapperArgs = [
  "--prefix" "PATH" ":" "${stdenv.lib.makeBinPath [ utillinux ]}"
];
```

**Comment by jtojnar** [comment]
0.7.1 has switched to pygobject3: https://github.com/whipper-team/whipper/blob/v0.7.1/CHANGELOG.md#v071-2018-10-23

**Comment by Moredread** [comment]
@jtojnar @Mic92: Done. Any idea how we can eliminate the patch for cdparanoia? Whipper expects it as cd-paranoia.

**Comment by jtojnar** [comment]
I actually prefer the patch approach to the wrapper – it ensures the programs are still necessary. That is actually why I introduced it. Though if the whipper maintainers prefer the wrapper method, they can feel free to change it.

**Comment by Moredread** [comment]
@jgeerds @rycee what is your opinion on this?

**Comment by Mic92** [comment]
Yes. It is too easy to miss a spot where a program was also called in my opinion.

**Pull request merged**